### DISCUSSION TITLE
creating new event with Node-Red

### DISCUSSION THREAD

**Discussion by Maximus48p** [blakeblackshear/frigate#8148] [Ask A Question] [answered]
I have setup with Home Assistant, Node-Red and Frigate and i want to create an new event when someone is pressing the doorbell.
Like the example in:
https://deploy-preview-6262--frigate-docs.netlify.app/integrations/api#post-apieventscamera_namelabelcreate

I'm using Node-Red node 'http request' for doing this with underneath POST url:
http://192.168.1.10:5000/api/events/camera1/doorbell/create

But when i do this POST message i always get a 404 error that the url does not exist.
When i do a GET request like 'http://192.168.1.10:5000/api' i don't get an error.

![image](https://github.com/blakeblackshear/frigate/assets/38084877/2516680f-4185-4ed5-88ca-b5221f932405)

What am i doing wrong here?

**Comment by NickM-27** [comment]
what version of frigate are you running

**Comment by Maximus48p** [comment]
0.12.1-367D724

**Reply by NickM-27** [reply, marked as answer]
this feature is currently only available in 0.13 beta

**Reply by Maximus48p** [reply]
aaaahhhhh.... and do i also need frigate+?

**Reply by NickM-27** [reply]
no, frigate+ is for training tuned object detection models, it is completely unrelated to the features that frigate itself offers

**Reply by Maximus48p** [reply]
ok, thank you very much for your quick and helpfull reply. I will upgrade to 0.13 beta. thx!

**Reply by NickM-27** [reply]
please be sure to read the release notes and breaking changes

**Comment by Maximus48p** [comment]
Works in 0.13.0-beta2 when using Postman, but not when i use Node-Red. Getting error: 
{"message":"An unknown error occurred: 'int' object has no attribute 'get'","success":false}

**Reply by NickM-27** [reply]
the node red request is incorrect then and sending some values with the incorrect type / format

**Reply by Maximus48p** [reply]
strange... i got it working by using a different node from 'node-red-contrib-http-request'
thx again!

**Reply by gamer06** [reply]
> strange... i got it working by using a different node from 'node-red-contrib-http-request' thx again!

hello. Were you able to create an event through node red? and then send it via Telegram for example? Can you share the flow?
Thanks
